package-lock.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I am coping vertbatim the comment I made on an earlier PR #130, @amanvats PTAL at this. If an issue is not already raised for this, then lets create one. I see more and more people putting this as a part of commit.  A short fix can you in your PR add `package-lock.json` in the `gitignore`. That will prevent future addition of this file until a decision is reached :)

> @isunitha98selvan IMHO the decision whether to use a package-lock.json instead of package,json should be first tracked in a separate issue, and them this file should be added accordingly. We need to weight in the pros/cons.
Please remove this file and create an issue to discuss addition of this file, and then add it via a separate PR based on the decision reached. Also, since this is a feature of npm 5 if I know correctly, we need to specify somewhere in the doc to make sure contributors are at npm5+ version. Please look into this.